jck/jck.mk

```diff
@@ -60,7 +60,13 @@ endif
 OTHER_OPTS=
 # if JDK_IMPL is openj9 or ibm
 ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
- OTHER_OPTS=-Xtrace:maximal=all{level2}
+ OTHER_OPTS=" -Xtrace:maximal=all{level2} -Xfuture "
+ ifeq ($(OS),OS/390)
```

Comment:
On line 64, please fix indentation. The line is missing a space at the beginning.